### DISCUSSION TITLE
Preferential education for teachers

### DISCUSSION THREAD

**Discussion by Dongyouxia** [community/community#114330] [GitHub Education]
### Hi everyone,

Is there any teacher who prompts the email address to use the student's email address when selecting the teacher registration?
![1](https://github.com/community/community/assets/53158423/5de43148-1637-487f-b41e-ac60a3dbfad1)

**Comment by samus-aran** [comment]
Hi @Dongyouxia  👋

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

You should provide evidence to support if your school has additional academic domains, has changed its academic email issuing policy, or does not provide academic emails. Reach out to your institution's IT team for this. 

Hope this helps! Let us know how you get along 👍

**Reply by Dongyouxia** [reply]
Hello, I have submitted the work order, and also attached a photo proof, but I have not been processed, is there anything else I need to provide? Work Order Number: # 2729312.What should I do, please let me know, thank you.
